src/platform/site-wide/sass/shame.scss

```diff
@@ -571,3 +570,13 @@ img[data-srcset] {
   border-top-left-radius: 0;
   border-bottom-left-radius: 0;
 }
+
+.bluecard {
```

Comment:
I thought we went with sco-card?

Reply:
Pushed the changes with `sco-card` css selector.